magento/magento2
# Fixed configurable product price indexer issue

### Description (*)

Configurable products price indexer save min_price in negative number i.e. -0.010000.

### Related Pull Requests


### Related Issues (if relevant)

1. Fixes #24535
2. Fixes #15609
3. Fixes #14443

### Manual testing scenarios (*)

1. Migrate Magento 1.9.4.2 > Magento 2.3.4
2. After the migration process - run full re-indexer
3. Then check the `catalog_product_index_price` as per the screenshot attached the min_price is incorrect.
![Selection_007](https://user-images.githubusercontent.com/24892273/74226147-0a754980-4ce2-11ea-88a0-d902d6e1ed5a.png)


### Questions or comments


### Contribution checklist (*)
